hub/client/config.py

```diff
@@ -21,3 +21,5 @@
 DEFAULT_REQUEST_TIMEOUT = 170
 
 HUB_AUTH_TOKEN = "HUB_AUTH_TOKEN"
+
+CONVERT_GRAYSCALE_IMAGES_TO_3D = True
```

Comment:
this constant will lead to lots of if cases, instead modify the [htype definitions](https://github.com/activeloopai/Hub/blob/jr-grayscale/hub/htype.py) for a more modular solution

Reply:
something like "require_3d"

Reply:
if we use the require_3d flag, then any sample added should be casted to 3D. this seems like a more permanent and extendible solution. maybe we should discuss this? this could have pretty broad implications.

Reply:
Constant has been removed.